compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.h

```diff
@@ -476,7 +476,40 @@ struct __sanitizer_cmsghdr {
   int cmsg_level;
   int cmsg_type;
 };
-#else
+#  elif SANITIZER_MUSL
+struct __sanitizer_msghdr {
+  void *msg_name;
+  unsigned msg_namelen;
+  struct __sanitizer_iovec *msg_iov;
+#    if __LONG_MAX > 0x7fffffff && __BYTE_ORDER == __BIG_ENDIAN
+  int __pad1;
+#    endif
+  int msg_iovlen;
+#    if __LONG_MAX > 0x7fffffff && __BYTE_ORDER == __LITTLE_ENDIAN
+  int __pad1;
```

Comment:
do you have a link to declaration?

Reply:
You mean for `__pad1`?
It is not a macro.